bothan-api/server-cli/src/commands/start.rs

```diff
@@ -0,0 +1,185 @@
+use std::fs::{create_dir_all, File};
+use std::io::Read;
+use std::path::PathBuf;
+use std::str::FromStr;
+
+use anyhow::Context;
+use clap::Parser;
+use reqwest::header::{HeaderName, HeaderValue};
+use semver::VersionReq;
+use tonic::transport::Server;
+
+use bothan_api::api::CryptoQueryServer;
+use bothan_api::config::ipfs::IpfsAuthentication;
+use bothan_api::config::manager::crypto_info::sources::CryptoSourceConfigs;
+use bothan_api::config::AppConfig;
+use bothan_api::proto::query::query_server::QueryServer;
+use bothan_api::VERSION;
+use bothan_binance::BinanceWorkerBuilder;
+use bothan_coingecko::CoinGeckoWorkerBuilder;
+use bothan_core::ipfs::{IpfsClient, IpfsClientBuilder};
+use bothan_core::manager::CryptoAssetInfoManager;
+use bothan_core::registry::Registry;
+use bothan_core::store::SharedStore;
+use bothan_core::worker::AssetWorkerBuilder;
+
+#[derive(Parser)]
+pub struct StartCli {
```

Comment:
implement --help dee mai?

Reply:
help is automatically implemented by clap